spaces/domains.json

```
@@ -850,5 +850,6 @@
   "vote.posichain.org": "posichaindao.eth",
   "vote.hausofprime.com": "hausofprime.eth",
   "gov.ohgeez.org": "levx.eth",
-  "snapshot.panvala.com": "panvala.eth"
+  "snapshot.panvala.com": "panvala.eth",
+  "vote.innercircle.finance": "innercirclefinance.eth",
```

Please remove the comma at the end of your domain line.
```suggestion
  "vote.innercircle.finance": "innercirclefinance.eth"
```